app/controllers/event_controller.rb

```diff
@@ -0,0 +1,52 @@
+class EventController < ApplicationController
+  before_action :session?
+  def index
+    @new_list = Event.where('events.event_date >= ?', DateTime.current.beginning_of_day)
+    @old_list = Event.where('events.event_date < ?', DateTime.current.beginning_of_day)
+    @users = User.all
+    @attendee = Attendee.where(user_id: cookies[:userid])
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = Event.new(user_params)
+    @event.user_id = cookies[:userid]
+    if @event.save
+      redirect_to events_Home_path
+    else
+      format.html { render :new }
+      format.json { render json: @event.errors, status: :unprocessable_entity }
+    end
+  end
+
+  def all
+    @user = User.find(cookies[:userid]).events
+  end
```

Comment:
This shouldn't be in the events_controller. 
You are missing the `index` method here where you should display the past and upcoming events